### DISCUSSION TITLE
I would like to connect a controller, a MUI text field and react-number-format. Why do I get back an undefined value?

### DISCUSSION THREAD

**Discussion by REDLIGHTFILMS** [react-hook-form/react-hook-form#3061] [Q&A]
I have almost finished the code. But unfortunately it prints the MUI text field as undefined value when output via react-hook-form... What can be the reason for this?

My code: https://codesandbox.io/s/intelligent-carson-hxyk5

**Comment by bluebill1049** [comment]
?? steps to reproduce?
